demos/test_vlc_net/scripts/watch_dog.py

```
@@ -0,0 +1,60 @@
+            self.end_headers()
+            self.wfile.write(log_handler.content.encode())
+        else:
+            self.send_response(404)
+            self.end_headers()
+
+with socketserver.TCPServer(("", PORT), CustomHandler) as httpd:
+    print(f"Serving at port {PORT}")
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        pass
```

Ruff

Use `contextlib.suppress(KeyboardInterrupt)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(KeyboardInterrupt)`

(SIM105)